packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -258,4 +283,15 @@ export default defineComponent({
     }
   }
 }
+
+.imprint-footer {
+  background-color: var(--oc-color-background-hover);
+  margin-left: -8px;
```

Comment:
please use the respective spacing css variable. otherwise the margin-correction breaks if someone changes their spacing css values.

Reply:
yes you are right, didn't know you could calc with vars